nixos/modules/services/databases/mysql.nix

```diff
@@ -375,6 +375,18 @@ in
           fi
         '';
 
+        script = ''
+          # https://mariadb.com/kb/en/getting-started-with-mariadb-galera-cluster/#systemd-and-galera-recovery
+          if test -n "''${_WSREP_START_POSITION}"; then
+            if test -e "${cfg.package}/bin/galera_recovery"; then
+              VAR=`cd ${cfg.package}/bin/..; ${cfg.package}/bin/galera_recovery`; [ $? -eq 0 ] && export _WSREP_START_POSITION=$VAR || exit 1
```

Comment:
```suggestion
              VAR=$(cd ${cfg.package}/bin/..; ${cfg.package}/bin/galera_recovery); [[ $? -eq 0 ]] && export _WSREP_START_POSITION=$VAR || exit 1
```

Reply:
Thanks for the suggestion @SuperSandro2000. I have applied, tested, and pushed it :+1: